sdks/techbooks-ts/src/lib/security.ts

```diff
@@ -23,8 +23,8 @@ export class SecurityError extends Error {
     }
     static unrecognizedType(type: string): SecurityError {
         return new SecurityError(
-            SecurityErrorCode.UnrecognisedSecurityType,
-            `Unrecognised security type: ${type}`
+            SecurityErrorCode.UnrecognizedSecurityType,
+            `Unrecognized security type: ${type}`
```

Comment:
There's really no reason for this human readable spelling. The majority of this repository favors the en-US (`-ize` family) ignoring the `SecurityErrorCode.UnrecognisedSecurityType` enum itself which may not be fixable.